site/styles/_footer.scss

```diff
@@ -49,6 +49,34 @@ footer {
   }
 }
 
+.social-media-icons {
```

Comment:
This has global scope so there is potential for it to bugger up social media icons that exist elsewhere.  I would stick the word "footer" in there some how.

Reply:
Fixed